src-tauri/src/analytics/algo_result_enums/diff_cen.rs

```diff
@@ -2,7 +2,7 @@ use std::collections::HashMap;
 
 #[derive(Debug)]
```

Comment:
This result is serialized on the main branch as of now (`#[derive(Debug, Serialize, Deserialize)]`), which might help with typescript errors. I don't think we can pass the data from the backend to the front via command without serializing.

Reply:
Yeah I will pull from the main soon.

Reply:
Typescript errors come from the accordion elements. IDK what's going on with those